.github/workflows/benchmarks.yml

```diff
@@ -30,6 +30,7 @@ jobs:
           -DCMAKE_C_COMPILER=${{matrix.compiler.c}}
           -DCMAKE_CXX_COMPILER=${{matrix.compiler.cxx}}
           -DUMF_BUILD_BENCHMARKS=ON
+          -DUMF_BUILD_BENCHMARKS_MT=ON
```

Comment:
Do we need a separate option for this? I think 'benchmarks' also include 'mt benchmarks'.

Reply:
I think it was added because MT bench requires C++. So either this new option or we enable C++ for all benchmarks, I guess

Reply:
If so, we could either leave this option or use C++ in all benchmarks. For me it could stay - just please add it to the README

Reply:
Done.